### ISSUE TITLE
[BUG] Pie Chart Component throws warning each element should have unique "key" prop

### ISSUE THREAD

**Issue by Abhijit-Dixit** [Abhinandan-Kushwaha/react-native-gifted-charts#224]
**Describe the bug**
When the data prop of Pie Chart Component is updated it throws the warning 
```
Warning: Each child in a list should have a unique "key" prop.
Check the render method of `PieChart`. See https://reactjs.org/link/warning-keys for more information.
```

**To Reproduce**
1. Create PieChart component
2. Data prop changes when state updates
3. Update data to see the error

**Expected behavior**
Do not throw the warning

**Screenshots**
![image](https://user-images.githubusercontent.com/41284892/172704804-27420366-aed4-4961-8c80-ea3d91570595.png)

 - Device: Samsung M21, Asus Zenfone Max Pro M1
 - OS: android 9
 - Version of the library being used 1.2.17, Expo version 5.3.0

**Additional context**
 
Although after dismissing the warning the component functions fine. When looking into the source code of Pie Chart Component I noticed that several array.map have been used without any keys, which might be causing this.

**Comment by Abhinandan-Kushwaha** [comment]
Hi @Abhijit-Dixit 👋
The issue is fixed in version `1.2.33`🎉

**Comment by Abhijit-Dixit** [comment]
Hi @Abhinandan-Kushwaha 
Thanks for the update!

**Comment by rrdesignweb** [comment]
Hi there I am also having this issue / warning message - `Check the render method of "PieChartMain"`

version `"react-native-gifted-charts": "^1.2.42",` 

thanks

**Comment by Abhinandan-Kushwaha** [comment]
Hi @rrdesignweb 
This is fixed in version `1.3.1`
Please use the latest version
